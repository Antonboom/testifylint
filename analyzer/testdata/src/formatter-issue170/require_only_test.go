package formatterissue170

import (
	"fmt"
	"testing"

	"github.com/stretchr/testify/require"
)

func TestFormatter(t *testing.T) {
	require.True(t, false, fmt.Sprintf("expected %v, got %v", true, false)) // want "formatter: remove unnecessary fmt\\.Sprintf"
}
