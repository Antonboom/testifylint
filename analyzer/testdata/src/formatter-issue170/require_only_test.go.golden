package formatterissue170

import (
	"fmt"
	"testing"

	"github.com/stretchr/testify/require"
)

func TestFormatter(t *testing.T) {
	require.True(t, false, "expected %v, got %v", true, false) // want "formatter: remove unnecessary fmt\\.Sprintf"
}
